README.md

```diff
@@ -250,6 +236,36 @@ $ truckersmp-cli --wine --prefixdir "/path/to/prefix/pfx" start ets2mp
     * When using Proton, wined3d can be used by specifying `--use-wined3d`, but it's not recommended because this is slower than OpenGL.
 * Used only when `-d` or `--enable-d3d11` is specified.
 
+## Configuration file
+
+It's possible to set third party programs in the configuration file that will get started together with the game. The format is similar to the INI format known from Windows.
+
+The file should contain sections following the format `[thirdparty.gamemode.programname]` (one game mode) or `[thirdparty.programname]` (all game modes).
+
+Each section requires one key-value-pair namely `executable = /path/to/my.exe` where the path is either an absolute unix path (`/path/to/my.exe`), a relative unix path (`program/my.exe`) to `$XDG_DATA_HOME/truckersmp-cli/` or an absolute windows path (`C:\Program Files (x86)\program\my.exe`).
+
+Additionally a `wait = seconds` key-value-pair can set a minimal waiting time before the game will be started so third party programs have enough time to fully start up and log in.
+
+### Example configuration file
+
+~~~ ini
+# Start Trucky together with all game modes (ats, atsmp, ets2, ets2mp)
+# and wait at least 45 seconds before starting the game
+[thirdparty.trucky]
+executable = /home/user/Downloads/trucky/trucky.exe
+wait = 45
+
+# Start TrucksBook together with ets2mp while waiting at least 30 seconds
+# before starting the game
+[thirdparty.ets2mp.trucksbook]
+executable = C:\Program Files (x86)\TrucksBook Client\TB Client.exe
+wait = 30
+
+# Allow using discord rich presence in singleplayer of ets2
+[thirdparty.ets2.richpresence]
+executable = wine-discord-ipc-bridge/winediscordipcbridge.exe
```

Comment:
wine-discord-ipc-bridge should not be specified here. See #219.

```suggestion
```

Reply:
That isn't merged yet…

> [PR 219 would be included in 0.8.0 too](https://github.com/truckersmp-cli/truckersmp-cli/pull/217#issuecomment-894597069)

Reply:
Imagine that some users actually save the setting *before 0.7.0, and after merging that PR*. The example would be *confusing* for users.

Reply:
I guess it's better to use your time to review #219 than to waste your time to discuss such thing: Do you want to also include #219 into 0.7.0?

Reply:
If it's ready, yes.